Lib/test/test_urllib2_localnet.py

```diff
@@ -315,14 +316,14 @@ def test_basic_auth_httperror(self):
         self.assertRaises(urllib.error.HTTPError, urllib.request.urlopen, self.server_url)
 
 
+@hashlib_helper.requires_hashdigest("md5", openssl=True)
```

Comment:
CPython doesn't specify `openssl=True`, is there a reason why this is needed?

Reply:
Which CPython version are you referencing? [`openssl=True` has been there since CPython 3.9.](https://github.com/python/cpython/blob/3.9/Lib/test/test_urllib2_localnet.py#L319)

Reply:
You're right. I accidentally compared it with CPython master branch.

Reply:
If it's removed in CPython's `main` branch, then we'll deal with it when CPython 3.11 gets released.